modules/apps/translation/translation-test/src/testIntegration/java/com/liferay/translation/manager/test/TranslationManagerTest.java

```diff
@@ -83,6 +94,120 @@ public void testGetXLIFFFile() throws Exception {
 		_testGetXLIFFFile("test-journal-article.xlf", _MIMETYPE_XLIFF_2_0);
 	}
 
+	@Test
+	public void testProcessXLIFFTranslation() throws Exception {
+		File xliff12File = _translationManager.getXLIFFFile(
```

Comment:
I would just name it xliffFile. @robertoDiaz, can we split the test in 4 cases instead of a big one?